### PR TITLE
Fix testing and remove warnings for r2d2m2cp

### PR DESCRIPTION
**Pull request by ferrine**

### CODE REVIEW

File: pymc_experimental/tests/distributions/test_multivariate.py [hunk 1]
```diff
@@ -127,9 +125,63 @@ def test_init(
         assert "beta" in model.named_vars
         assert ("beta::r2" in model.named_vars) == (r2_std is not None), set(model.named_vars)
         # phi is only created if variable importance is not None and there is more than one var
+        assert np.isfinite(sum(model.point_logps().values()))
```
Comment (ricardoV94) [inline pymc_experimental/tests/distributions/test_multivariate.py:128]:
```suggestion
        assert np.isfinite(model.compile_logp()(model.initial_point()))
```

File: pymc_experimental/tests/distributions/test_multivariate.py [hunk 2]
```diff
@@ -127,9 +125,63 @@ def test_init(
...
+        assert not np.isnan(beta.eval()).any()
+        assert eps.eval().shape == output_std.shape
+        assert beta.eval().shape == input_std.shape
+        # r2 rv is only created if r2 std is not None
+        assert "beta" in model.named_vars
+        # phi is only created if variable importance is not None and there is more than one var
         assert ("beta::phi" in model.named_vars) == (
             "variables_importance" in phi_args or "importance_concentration" in phi_args
         ), set(model.named_vars)
+        assert np.isfinite(sum(model.point_logps().values()))
```
Comment (ricardoV94) [inline pymc_experimental/tests/distributions/test_multivariate.py:157]:
```suggestion
        assert np.isfinite(model.compile_logp()(model.initial_point()))
```

**Review by ricardoV94 — APPROVED** [review]

**Comment by ferrine** [comment]
checks got lost

**Comment by ricardoV94** [comment]
Yeah it's odd. Can you push something to see if it triggers them?

**Pull request merged**